### ISSUE TITLE
Pivot table datetime representation is ugly

### ISSUE THREAD

**Issue by tonyjiangh** [getredash/redash#2360]
### Issue Summary
![image](https://user-images.githubusercontent.com/16881036/36876899-b94341c4-1dfa-11e8-875e-d43460a01fd1.png)

This looks ugly. Since we're formatting date in other visualization it feels better to do something about this as well.
I do realized that we are using pivottable.js. Not sure yet if this is their feature.

### Technical details:

* Redash Version: v4.0.0-beta (570187fc1f96d336ed63088c83efc1c56298a7cb)
* Browser/OS: Firefox
* How did you install Redash: Docker

**Comment by arikfr** [comment]
Related: #216.

**Comment by deecay** [comment]
Doing something like this in pivot/index.js can change the "datetime" column representation.

`moment.fn.toString = function () { return this.format('ddd MMM DD YYYY'); };`

This will not break the sorting in the pivottable. It just alters the way dates are represented. Related code [here](https://github.com/nicolaskruchten/pivottable/blob/d67aef3f11aa5a5c10275f6726e78f616066d55f/src/pivot.coffee#L504).

If we want to fine tune the format for individual columns, such as columA as date, and columB as timestamp, then we probably need some configuration UI like the one for table viz. WDYT?

**Comment by arikfr** [comment]
@gabrieldutra did this change now that we're using the React lib?

**Comment by gabrieldutra** [comment]
> did this change now that we're using the React lib?

Yes, now it uses the same `formatDate` and `formatDatetime` that are used in Filters and consider the organization settings

**Comment by arikfr** [comment]
And do you know if it uses the moment object for sorting?

**Comment by gabrieldutra** [comment]
> And do you know if it uses the moment object for sorting?

Certainly it doesn't, the `react-pivottable` lib only accepted strings, so I had to convert them. So the sorting should be 100% on the formatted dates.

**Comment by arikfr** [comment]
Bummer.

**Comment by inggilove** [comment]
> Yes PLEASE make that UI change. It is ugly right now. We use Big Query. I get around it by FORMAT_DATE in standard SQL or STRFTIME_UTC_USEC("2016-10-20", "%m/%d/%Y") in legacy sql.

Thanks!
in Presto SQL, I use like below, it is working!

select date_format(date(createdat_date), '%m-%d-%Y') as daily ...
